src/containers/ErrorBoundary/LayoutErrorFallback.tsx

```
@@ -0,0 +1,44 @@
+import { Header } from 'containers';
+
+import { Box, useMediaQuery } from '@mui/material';
+
+import { Error } from '@component';
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)